packages/generator/src/v2.0/UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst.ts

```diff
@@ -48,23 +49,25 @@ export function __UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst(
   const objectDefIdentifier = interfaceDef.getDefinitionIdentifier(v2);
 
   const parents = definition.implements?.map(p => {
-    invariant(
-      ontology.interfaceTypes[p] != null,
-      `Expected to find a parent interface named ${p} in the ontology and did not.`,
-    );
-
-    const it = deleteUndefineds(
-      __UNSTABLE_wireInterfaceTypeV2ToSdkObjectDefinition(
-        ontology.requireInterfaceType(p, true).raw,
-        v2,
-      ),
-    );
-
-    return it;
+    const parent = ontology.requireInterfaceType(p, true);
+    if (parent instanceof EnhancedInterfaceType) {
+      const it = deleteUndefineds(
+        __UNSTABLE_wireInterfaceTypeV2ToSdkObjectDefinition(
+          parent.raw,
+          v2,
+        ),
+      );
+      return it;
+    }
   }) ?? [];
 
   const mergedProperties = { ...definition.properties };
   for (const parent of parents) {
+    if (parent == null) {
+      // came from a foreign type and we cannot merge properties yet
```

Comment:
I'm still a bit confused by this comment

Reply:
The current implementation assumes that we are generating everything in the `ontology.json`. We don't have the external objects in there because we are not generating them. But that means we cannot, at this part of the code, copy the extended interface's properties onto ourself. 

We can adjust this later to support external things being listed in the ontology.json but I don't have time for that right now.